test/TestApps/ToastNotificationsTestApp/main.cpp

```diff
@@ -378,6 +378,37 @@ bool VerifyToastExpiresOnReboot()
     return true;
 }
 
+bool VerifyShowToast()
```

Comment:
VerifyShowToast_Packaged

Reply:
IMHO there is no need to be that explicit. All unpackaged tests are marked as `_Unpackaged` and we call `RunTestUnpackaged` to run the test app as unpackaged. Anything else is a packaged-scenario test.